src/System.Text.Primitives/System/Text/Formatters/Utf16.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text.Formatters
+{
+    public static partial class Utf16
+    {
+        #region Common constants
+
+        private const char Colon = ':';
```

Comment:
Can you please explain how you decide what constants get moved to "common constants" here? Why is Colon here and why isn't Separator in Utf16.Integer?